### PR TITLE
Add google oauth docs

### PR DESCRIPTION
**Pull request by mrajashree**

### CODE REVIEW

File: content/rancher/v2.x/en/admin-settings/authentication/google/_index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+_Available as of v2.3_
+
+If your organization uses G Suite for user authentication, you can configure Rancher to allow your users to login using their G Suite credentials.
+
+>**Prerequisites:**
+>
+>- You must have a [G Suite admin account](https://admin.google.com) configured.
+>- You must have "Admin SDK" API enabled for your G Suite domain. You can enable it using [these](https://support.google.com/a/answer/60757?hl=en) steps  
+Once enabled, this is what your GSuite domain's API screen would look like
```
Comment (prachidamle) [inline content/rancher/v2.x/en/admin-settings/authentication/google/_index.md]: nit: "this is how your.."

**Comment by mrajashree** [comment]
@prachidamle added a section on service account and why it's needed

File: content/rancher/v2.x/en/admin-settings/authentication/google/_index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+5. Click on **Create Key**, select JSON option,this will download a JSON file which you will provide as the service account credentials to Rancher  
+![Service account creation Step 3]({{< baseurl >}}/img/rancher/Google-svc-acc-step3-key-creation.png)  
+6. Using the Unique ID of this key, register it as an Oauth Client using the following steps
+	1.  Get the Unique ID of the key you just created. This is a numeric key, if it's not displayed in the list of keys right next to the one you created, you will have to enable it to be listed by doing the following  
+	![Service account Unique ID]({{< baseurl >}}/img/rancher/Google-Select-UniqueID-column.png)  
+	2.  `Unique ID` must be clicked, and then click on OK. This will add a column to the list of service account keys for Unique ID. Save the one listed for the service account you created. NOTE: This is a numeric key, not to be confused with the alpha numeric field `Key ID`  
+	3. Go to `Manage OAuth client access` page: [https://admin.google.com/AdminHome?chromeless=1#OGX:ManageOauthClients](https://admin.google.com/AdminHome?chromeless=1#OGX:ManageOauthClients)
```
Comment (sowmyav27) [inline content/rancher/v2.x/en/admin-settings/authentication/google/_index.md]: hyperlink has to modified here

File: content/rancher/v2.x/en/admin-settings/authentication/google/_index.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+4. Don't provide any roles on the **Service account permissions** page and click **Continue**  
+![Service account creation Step 2]({{< baseurl >}}/img/rancher/Google-svc-acc-step2.png)  
+5. Click on **Create Key**, select JSON option,this will download a JSON file which you will provide as the service account credentials to Rancher  
+![Service account creation Step 3]({{< baseurl >}}/img/rancher/Google-svc-acc-step3-key-creation.png)  
+6. Using the Unique ID of this key, register it as an Oauth Client using the following steps
+	1.  Get the Unique ID of the key you just created. This is a numeric key, if it's not displayed in the list of keys right next to the one you created, you will have to enable it to be listed by doing the following  
+	![Service account Unique ID]({{< baseurl >}}/img/rancher/Google-Select-UniqueID-column.png)  
+	2.  `Unique ID` must be clicked, and then click on OK. This will add a column to the list of service account keys for Unique ID. Save the one listed for the service account you created. NOTE: This is a numeric key, not to be confused with the alpha numeric field `Key ID`
```
Comment (sowmyav27) [inline content/rancher/v2.x/en/admin-settings/authentication/google/_index.md]: Can we have the step here which says something like "Check the box next to `Unique ID` and click on OK"?

File: content/rancher/v2.x/en/admin-settings/authentication/google/_index.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+4.  Step One tells how to add your domain to list of authorized domains. For detailed steps refer [this section]({{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/google/#adding-rancher-as-an-authorized-domain)
+
+5.  Step Two describes creation of OAuth credentials. For details on how to generate this file, refer [this]({{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/google/#creating-oauth2-credentials-for-rancher-server)
+
+6.  Step Three requires generation of Service Account credentials and registering the service account unique ID as an OAuth client in your GSuite account. Refer [this section]({{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/google/#creating-service-account-credentials) for detailed steps.
+
+7.	Click **Authenticate with Google**.
+
+8.	Click **Save**.
```
Comment (sowmyav27) [inline content/rancher/v2.x/en/admin-settings/authentication/google/_index.md]: We could remove this step. We do not do a `Save` after we click on Authenticate.

File: content/rancher/v2.x/en/admin-settings/authentication/google/_index.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+**Result:** Google authentication is configured.
```
Comment (sowmyav27) [inline content/rancher/v2.x/en/admin-settings/authentication/google/_index.md]: We could add this as a step, as it looks out of place on the Rancher docs.. 
Screen Shot 2019-09-26 at 4 23 41 PM

**Comment by deniseschannon** [comment]
@mrajashree make a new clean PR off of staging to add your single commit.

**Comment by mrajashree** [comment]
Closing in favor of https://github.com/rancher/docs/pull/1784
will address the pr review comments in this one in the other pr

**Pull request closed without merging**